apcd-cms/src/client/src/components/Extensions/EditExtensionModal/EditExtensionModal.tsx

```diff
@@ -0,0 +1,380 @@
+import React, { useState, useEffect } from 'react';
+import {
+  Modal,
+  ModalBody,
+  ModalFooter,
+  Button,
```

Comment:
If you  `import Button from core-components/Button` instead of Reactstrap, you will get the correct colors and styling. You'll need to change line 351 to something like 

```
                    <Button
                      type="primary"
                      attr="submit"
                      isLoading={isSubmitting}
                      onClick={() => setIsSuccess(false)}
                    >
                      Submit
                    </Button>
```